src/Security/Member.php

```diff
@@ -1123,6 +1123,16 @@ public static function set_title_columns($columns, $sep = ' ')
 
     //------------------- HELPER METHODS -----------------------------------//
 
+    /**
+     * Simple proxy method to get the Surname property of the member
+     *
+     * @return string
+     */
+    public function getLastName()
+    {
+        return $this->owner->Surname;
```

Comment:
This isn't an extension, it is the owner of the Surname property. Should be `$this->Surname` instead.

A unit test would also be good for this change.

Reply:
Dammit!! sorry @robbieaverill was being so careful to get the number of spaces to line up perfectly (that was why previous PR for this same thing got rejected) that I forgot to delete the `->owner`  part after copy/pasting from the extension in my own project. If I edit this in my fork of it, will it automatically get fixed or do I need to do something else?

Reply:
@Hels666 no problem =) I fixed this in #9226 already. For future reference, editing your fork will update pull requests like this but only until they're merged, then you'd need to raise a new pull request with your new changes.